src/bikeracks/templates/helpers.js

```diff
@@ -64,28 +64,7 @@ function getRack(state) {
         
 }
 
-// for testing purposes
```

Comment:
It's probably fine in this case, but you want to avoid making unrelated changes. Unrelated changes like this make it harder to review and harder to undo later if it turns out this PR is causing problems and needs to be reverted.

It's fine to have small PRs later that just clean up dead code or bad formatting, etc.

Reply:
Okay, I will save that kind of stuff for a separate PR next time.